filter.txt

```diff
@@ -7520,6 +7520,7 @@ news.appstory.co.kr##.header_banner
 news.appstory.co.kr##a[class^="banner_"]
 1412.rest##div[id^="plusad_"]
 1412.rest##.custom_code.board_top
+1412.rest#script[defer,onerror="adError();",onload="adSuccess();"]
```

Comment:
`#script` does not work on Mozilla Firefox with AdGuard Browser Extension.
And, Additional rule is not required because `||1412.rest/addons/*/adblockDetector.js` already exists in List-KR.

If you want to add a rule that block anti-adblock script in the website, you can use [`$$`](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#html-filtering-rules-1) or [`#%#//scriplet()`](https://github.com/AdguardTeam/Scriptlets).

Reply:
@green1052

Reply:
use 1412.rest#script[defer,onerror="adError();",onload="adSuccess();"] instaed

#%# rule?

Reply:
Please update List-KR and reload the website without your rule.

Reply:
https://github.com/List-KR/List-KR/pull/230#discussion_r822681624
Yes, please use `#%#//scriptlet()`, `$redirect` or `$$` instead of `1412.rest#script[defer,onerror="adError();",onload="adSuccess();"]` that cannot be used because it does not work.